public/app/plugins/datasource/mixed/MixedDataSource.ts

```diff
@@ -41,7 +41,7 @@ export class MixedDatasource extends DataSourceApi<DataQuery> {
     }
 
     // Build groups of queries to run in parallel
-    const sets: { [key: string]: DataQuery[] } = groupBy(queries, 'datasource.uid');
+    const sets: { [key: string]: DataQuery[] } = groupBy(queries, (item) => item.datasource?.uid || item.datasource);
```

Comment:
this is not supported anymore the datasource project is an object with uid and name on it, not a string. 

We migrate all instances of the string property a few versions ago.